modules/core/src/main/java/org/apache/ignite/internal/processors/service/GridServiceProxy.java

```diff
@@ -354,6 +358,37 @@ T proxy() {
         return proxy;
     }
 
+    /**
+     * Calls service method, measures and registers its duration.
+     *
+     * @param srvcProc Current service processor.
+     * @param srvc The service object.
+     * @param srvcName The service name.
+     * @param mtd Method to call.
+     * @param args Arguments for {@code mtd}.
+     */
+    private static Object measureServiceMethod(ServiceProcessorAdapter srvcProc, Service srvc, String srvcName,
```

Comment:
Rewrite a declaration, please, each param on new line, bracket and curly bracket also on new line

Reply:
Fixed.